src/Features/CSharp/Portable/LanguageServices/CSharpSymbolDisplayService.SymbolDescriptionBuilder.cs

```diff
@@ -101,6 +101,12 @@ protected override void AddAwaitableExtensionPrefix()
                 return SpecializedTasks.EmptyImmutableArray<SymbolDisplayPart>();
             }
 
+            protected override ImmutableArray<SymbolDisplayPart> ToMinimalDisplayParts(ISymbol symbol, SemanticModel semanticModel, int position, SymbolDisplayFormat format)
+                => CodeAnalysis.CSharp.SymbolDisplay.ToMinimalDisplayParts(symbol, semanticModel, position, format);
+
+            protected override string GetNavigationHint(ISymbol symbol)
+                => symbol == null ? null : CodeAnalysis.CSharp.SymbolDisplay.ToDisplayString(symbol, SymbolDisplayFormat.MinimallyQualifiedFormat);
```

Comment:
Had to thread through some information so that we can get doc comments for synthesized IDE symbols.